pyproject.toml

```
@@ -84,6 +83,7 @@ format-check = "ruff format --check {args}"
 
 [tool.hatch.envs.test]
 extra-dependencies = [
+  "pandas",                                    # Used in several components and pipeline tests
```

Let's enumerate the components explicitly:
```suggestion
  "pandas",                                    # Used in XLSXToDocument, CSVDocumentSplitter,  CSVDocumentCleaner, EvaluationRunResult and pipeline tests
```